runbotics/runbotics-orchestrator-ui/src/src-app/views/auth/RegisterPage/Register.page.tsx

```diff
@@ -105,7 +105,13 @@ const RegisterPage: FC = () => {
             .unwrap()
             .catch((error) => {
                 if (error.statusCode === 400) {
-                    enqueueSnackbar(error.message, { variant: 'error' });
+                    const customTitle = translate('Error.InviteCode.View.Title');
+                    const customMessage = translate('Error.InviteCode.View.Message');
+                    sessionStorage.setItem('errorTitle', customTitle);
+                    sessionStorage.setItem('errorMessage', customMessage);
+
+
+                    router.push('/customerror');
```

Comment:
I think 'error' would be clearer for users

Reply:
I know that for us *developers* it is well readable based on functionality, however this is url. You can share your opinion 😊